### DISCUSSION TITLE
Any way to set the port on a server?

### DISCUSSION THREAD

**Discussion by ghost** [scottoffen/grapevine#20] [Q&A] [answered]
Just a general question. Im struggling to find any infomation on how to configure the port my server runs on.

My code:

```cs
public APIServer()
{
    Port = 3000;
    rawServer = RestServerBuilder.UseDefaults().Build();
    RouteManager.RegisterToRouter(rawServer.Router);
    rawServer.UseCorsPolicy();
    rawServer.Start();
}
```

**Comment by ghost** [comment, marked as answer]
Nevermind :D I found out i could add Prefixes.

**Reply by scottoffen** [reply]
Hey, I just answered your other question over in issues! Good to see you here.
